.circleci/config.yml

```diff
@@ -540,6 +583,23 @@ jobs:
           enable-dashboard: << parameters.enable-dashboard >>
           suite: << parameters.suite >>
 
+  #Maybe have a move this to a image folder
+  docker-deploy:
+    docker:
+      - image: cimg/node:16.16.0
+    steps:
+      - set-dev-env-vars
```

Comment:
Why are you setting the dev envs in this step?

Reply:
In order to mitigate the following error.

https://app.circleci.com/pipelines/github/newjersey/navigator.business.nj.gov/10630/workflows/f08e5d91-6386-4aae-bb4d-f3cf812e2111/jobs/47431